### PR TITLE
Add additional reviewers to webrender repositories

### PR DESCRIPTION
**Pull request by jdm**
This seems like the state of the world right now.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/saltfs/590)

### CODE REVIEW

File: homu/files/cfg.toml [hunk 1]
```diff
@@ -165,10 +165,12 @@ secret = "{{ pillar["homu"]["web-secret"] }}"
...
     },
     {
-        "name": "webrender"
+        "name": "webrender",
+        "extra_reviewers": [ "jrmuizel", "vvuk" ]
     },
     {
-        "name": "webrender_traits"
+        "name": "webrender_traits",
+        "extra_reviewers": [ "jrmuizel", "vvuk" ]
```
Comment (emilio) [inline homu/files/cfg.toml]: This is probably useless because wr-traits now effectively lives in the wr repo.

**Comment by emilio** [comment]
I'd approve this myself, but I think I can't, so maybe r? @glennw

**Comment by jdm** [comment]
Rebased in #669.

**Pull request closed without merging**